### PR TITLE
fix issue with registry URL

### PR DESCRIPTION
**Pull request by julianduque**
Fixes an issue when the registry doesn't have a specified protocol

### CODE REVIEW

**Review by realtaraharris — COMMENTED** [review]
LGTM!

**Comment by realtaraharris** [comment]
last minute consideration that I thought of _after_ I hit the approval button:

```
% nscm config get registry
https://16e61233c10c947389b7.registry.nodesource.io/
% nscm config set registry "//localhost"
% nscm config get registry
//localhost
```

**Comment by julianduque** [comment]
@maxharris9 done! can you review again?

File: lib/tools.js [hunk 1]
```diff
@@ -284,3 +284,12 @@ function usageFilter (usage) {
   usage = usage.replace(/\[options]\s\[command]/, '[command] [options]')
   return brand + usage + additionalHelp
 }
+
+function normalizeRegistry (uri) {
+  if (!uri.match(/^http(s)\/\//)) {
+    uri = uri.replace(/\//g, '')
+    uri = `https://${uri}`
```
Comment (maxharris9) [inline lib/tools.js]: can we add a slash onto the end of that uri?

i.e., "uri = `https://${uri}/`"

**Review by realtaraharris — APPROVED** [review]

**Pull request merged**